src/rpc/net.cpp

```diff
@@ -460,6 +460,7 @@ UniValue getnetworkinfo(const UniValue& params, bool fHelp)
             "  \"subversion\": \"/Satoshi:x.x.x/\",     (string) the server subversion string\n"
             "  \"protocolversion\": xxxxx,              (numeric) the protocol version\n"
             "  \"localservices\": \"xxxxxxxxxxxxxxxx\", (string) the services we offer to the network\n"
+            "  \"localrelay\": true|false,              (bool) true if transaction relay is requested from peers\n"
```

Comment:
Possibly - What the flag is used for is to specify in the version message whether we want to receive transactions (so whether transactions should be relayed *to us*), as well as in the network code whether we accept transactions. That we don't relay transactions is also true, but follows from that.
Maybe the naming is confusing, but I don't know a better one either.